subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,192 @@
+== Identify the Dependency Version Selected
 
-Let's have a look at a concrete example.
-A project may request two different versions of the same dependency either as direct or transitive dependency.
+A project may request two different versions of the same dependency either directly or transitively.
 Gradle applies <<dependency_resolution.adoc#sec:version-conflict,version conflict resolution>> to ensure that only one version of the dependency exists in the dependency graph.
-In this example the conflicting dependency is represented by `commons-codec:commons-codec`.
+The following example introduces a conflict with `commons-codec:commons-codec`, added both as a direct dependency and a transitive dependency of JGit:
 
-.Declaring the JGit dependency and a conflicting dependency
 ====
 include::sample[dir="snippets/dependencyManagement/inspectingDependencies-dependencyInsightReport/groovy",files="build.gradle[tags=dependency-declaration]"]
 include::sample[dir="snippets/dependencyManagement/inspectingDependencies-dependencyInsightReport/kotlin",files="build.gradle.kts[tags=dependency-declaration]"]
 ====
 
-The dependency tree in a link:https://scans.gradle.com/[build scan] renders the selection reason (conflict resolution) as well as the origin of a dependency if you click on a dependency and select the "Required By" tab.
+The dependency tree in a link:https://scans.gradle.com/[build scan] shows information about conflicts.
+Click on a dependency and select the "Required By" tab to see the selection reason and origin of the dependency.
 
-.Dependency insight capabilities in a build scan
 image::dependency-management-dependency-insight-report-build-scan.png[]
 
-Every Gradle project provides the task `dependencyInsight` to render the so-called  _dependency insight report_ from the command line.
-Given a dependency in the dependency graph you can identify the selection reason and track down the origin of the dependency selection.
-You can think of the dependency insight report as the inverse representation of the dependency report for a given dependency.
+=== Dependency Insights
 
-The task takes the following parameters:
+Gradle provides the built-in `dependencyInsight` task to render a  _dependency insight report_ from the command line.
+Dependency insights only provide information about a single project at a time.
+Given a dependency, you can identify the selection reason and origin.
+
+`dependencyInsight` accepts the following parameters:
 
 `--dependency <dependency>` (mandatory)::
-Indicates which dependency to focus on.
-It can be a complete `group:name`, or part of it.
-If multiple dependencies match, they are all printed in the report.
+The dependency to generate an insight report for.
+You can supply a complete `group:name`, or part of it.
+If multiple dependencies match, Gradle generates a report for each matching dependency.
```

Comment:
It is not a report per dependency, it is a single report which covers multiple dependencies.

Reply:
Is this more accurate?

```suggestion
If multiple dependencies match, Gradle generates a report covering all matching dependency.
```

Reply:
Yes